engine/src/operators/GroupBy.cpp

```diff
@@ -708,6 +708,25 @@ auto groupby_without_aggregations_and_sample(Context * context,
 	return std::make_pair(std::move(grouped_table), std::move(partitionPlan));
 }
 
+std::string get_agg_str(AggregateKind kind){
+	switch(kind){
+		case SUM:
+		case SUM0:
```

Comment:
This is not correct. SUM0 is a different kind of operator than SUM. We had at one point implemented both of these.

Reply:
SUM0 is a different kind of operator, yes. This is only used for setting column names, in which case I think we can just stick to SUM, since most users would not explicitly use SUM0 and it may just be confusing.

Reply:
Yes SUM0 is its own operator. This is used for setting the column names

Reply:
@rommelDB there is already a function for this purpose. Its `aggregator_to_string` and its in CalciteExpressionParsing. Please use that function.

Reply:
Fixed.